File: Projects/iOS/TestLab/DetailViewController.m
//
//  DetailViewController.m
//  Testlab
//
//  Created by SalmoJunior on 1/18/16.
//  Copyright © 2016 CI&T. All rights reserved.
//

#import "DetailViewController.h"

@interface DetailViewController ()

@property (weak, nonatomic) IBOutlet UILabel *resultLabel;

@end

@implementation DetailViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    
    self.resultLabel.text = [NSString stringWithFormat:@"Ok captain %@, you got it!", self.name];
}

@end

File: Projects/iOS/TestLab/DetailViewController.swift
//
//  DetailViewController.swift
//  TestLab
//
//  Created by SalmoJunior on 1/6/16.
//  Copyright © 2016 CI&T. All rights reserved.
//

import UIKit

class DetailViewController: UIViewController {

    @IBOutlet weak var resultLabel: UILabel!
    var name: String!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func viewWillAppear(animated: Bool) {
        super.viewWillAppear(animated)
        
        title = "Detail"
        resultLabel.text = "Ok captain \(name), you got it!"
    }

}
